### ISSUE TITLE
[BUG] range_tool broken on inverted axes

### ISSUE THREAD

**Issue by Connossor** [bokeh/bokeh#9381]
#### ALL software version info (bokeh, python, notebook, OS, browser, any other relevant packages)

bokeh.version: 1.3.4
Python version: 3.7.3
OS: GNU/Linux x86_64
Browser: Google Chrome 77.0.3865.90
JupyterLab version: 1.1.3

#### Description of expected behavior and the observed behavior

When a RangeTool is used with an inverted axis (i.e. a y axis in which the data increase downwards), the range tool is unresponsive.

In this example, the RangeTool should allow a user to control the zoom of the figure on the right, but the shaded area on the left cannot be moved up or down.

However, when a user pans the plot on the right, the RangeTool does still move up and down as expected.

The bug seems to only appear when the Range1d is inverted.

#### Complete, minimal, self-contained example code that reproduces the issue

```
from bokeh.models.ranges import Range1d
from bokeh.layouts import row
from bokeh.models import RangeTool
from bokeh.plotting import figure, show

xx = [20,22,21]
yy = [2,3,4]
kwgs = dict(plot_height=300, plot_width=300)

overview_rng = Range1d(5, 0)   # Range is inverted, higher number first
zoomed_rng = Range1d(4, 2)

fig_overview = figure(y_range=overview_rng, **kwgs)
fig_overview.line(xx, yy)

range_tool = RangeTool(y_range=zoomed_rng)
fig_overview.add_tools(range_tool)

fig_zoomed = figure(y_range=zoomed_rng, **kwgs)
fig_zoomed.line(xx, yy)

show(row([fig_overview, fig_zoomed]))
```

#### Screenshots or screencasts of the bug in action

![range_tool_inverted](https://user-images.githubusercontent.com/22548039/68316276-1a610a00-00b1-11ea-9707-d2e9e6e06372.gif)

**Comment by bryevdv** [comment]
Thanks for the detailed report @Connossor

**Comment by ncvangilse** [comment]
On bokeh 2.4.2, it is now possible to move the range_tool, but not drag the edges (changing only one end of the range).

**Comment by ccostadv** [comment]
Any update on this? Still broken on 2.4.3 and 3.0.3.

**Comment by mattpap** [comment]
This is fixed in PR #12468.
